src/actions/renderActions.js

```diff
@@ -1,35 +1,36 @@
 
```

Comment:
What I was suggesting to you guys yesterday is more something along the lines of 
export const SET_VIEW = 'SET_VIEW';

and then one function setView which provides all the info necessary for setting the view in the action.

Reply:
but you don't need to address this in this PR.